test/integration/connect/envoy/run-tests.sh

```diff
@@ -15,7 +15,7 @@ DEBUG=${DEBUG:-}
 XDS_TARGET=${XDS_TARGET:-server}
 
 # ENVOY_VERSION to run each test against
-ENVOY_VERSION=${ENVOY_VERSION:-"1.27.0"}
+ENVOY_VERSION=${ENVOY_VERSION:-"1.28.4"}
```

Comment:
It looks like the jump here is due to this update being missed in the past. Is that correct?

Reply:
Yes I am assuming so